src/components/Common/SelectRecordListFields.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div>
+    <b-button @click="showModal=true"
```

Comment:
properties that start with @ should be last in te prop list. 
Also Every prop on new line please